### DISCUSSION TITLE
How to pass optimization flags?

### DISCUSSION THREAD

**Discussion by nk9** [cloudinary-community/next-cloudinary#202] [General]
I would like to send requests to Cloudinary which include image optimization flags. In particular, I want to pass the [`fl_keep_iptc` flag](https://cloudinary.com/documentation/image_optimization) so that I can get the IPTC data from a scaled image.

I don't see a way to pass that in the current library. Am I missing it?

**Comment by colbyfayock** [comment]
@nk9 currently not a way - I'll work on getting this added!

**Reply by nk9** [reply]
Thank you very much!

**Reply by colbyfayock** [reply]
hey @nk9 this is now available

one issue i ran into was you can't use quality="auto" with that specific flag

I added the ability to set `quality="default"` which turns it off, as by default, the component opts in to `quality="auto"`

https://next-cloudinary.spacejelly.dev/components/cldimage/examples#flags

**Reply by nk9** [reply]
Thanks very much for the quick work on this, much appreciated! Glad everyone can benefit from this now.
